index.html

```diff
@@ -0,0 +1,671 @@
+            <a class="title-link" href="#"> The Sun Has Been Spotless for 28 Days Straight Despite New Solar Cycle</a>
+          </article>
+        </div>
+      </div>
+    </section>
+    <section class="category mt-2 mb-4">
+      <div class="big-red-title">Sports</div>
+      <div class="row">
+        <div class="col-sm-12 col-md-4">
+          <img src="img/sports-1.jpg" alt="">
+          <a class="title-link" href="#"> NBA Playoffs: How to Watch LA Lakers vs Denver Nuggets on TV and Online</a>
+        </div>
+        <div class="col-sm-12 col-md-4">
+          <img src="img/sports-1.jpg" alt="">
+          <a class="title-link" href="#"> NBA Playoffs: How to Watch LA Lakers vs Denver Nuggets on TV and Online</a>
+        </div>
+        <div class="sidebar col-sm-12 col-md-4 d-flex flex-column">
+          <article class="d-flex flex-row">
+            <img class="img-fluid" src="img/sports-side-1.jpg" alt="">
+            <a class="title-link" href="#"> Joe Burrow Sets Series of New NFL Records as Bengals Lose to Browns on TNF</a>
+          </article>
+          <article class="middle-bar d-flex flex-row">
+            <img class="img-fluid" src="img/sports-side-1.jpg" alt="">
+            <a class="title-link" href="#"> Joe Burrow Sets Series of New NFL Records as Bengals Lose to Browns on TNF</a>
+          </article>
+          <article class="d-flex flex-row">
+            <img src="img/sports-side-1.jpg" alt="">
+            <a class="title-link" href="#"> Joe Burrow Sets Series of New NFL Records as Bengals Lose to Browns on TNF</a>
+          </article>
+        </div>
+      </div>
+    </section>
+
+    <section class="combo fluid-container mt-4">
+      <section class="row ml-3">
+        <section class="subscribe w-lg-100 col-lg-7 col-md-12 col-sm-12">
+          <span class="red-title"> SUBSCRIBE </span>
+          <div class="subscribe-box d-flex">
+            <img alt="Article Image" src="img/subscribe.png">
+            <div class="subscribe-content fluid-container d-flex">
+              <p>In-depth Stories You Don't See Elsewhere About Topics You Want to Read!</p>
+              <p>All Access + Weekly Delivery.</p>
+              <p>Choose A Membership That's Perfect for You!</p>
+              <p><button class="red-button"> SUBSCRIBE </button></p>
+            </div>
+          </div>
+        </section>
+        <section class="newsletter w-lg-100 col-lg-5 col-md-12 col-sm-12">
+          <span class="red-title"> NEWSLETTER </span>
+          <div class="newsletter-box d-flex">
+            <img alt="Article Image" src="img/newsletter.png">
+            <div class="newsletter-content d-flex justify-content-start">
+              <h3>The Starting 5</h3>
+              <p>See why nearly a quarter of a million subscribers begin their day with the Starting 5.</p>
+              <input type="email" name="email" id="email">
+              <p><button class="red-button"> SUBSCRIBE </button></p>
+            </div>
```

Comment:
[OPTIONAL] The image seems to be stretched out here. You can use CSS properties like `object-fit` to fix such issues.

[OPTIONAL] The newsletter box doesn't seem to be utilizing the entire height of its container. Kindly fox it.

[OPTIONAL] Add padding between the subscribe button and the input.